Documentation/networking.md

```diff
@@ -6,16 +6,9 @@
 
 By default, `rkt run` will start the pod with host networking.
 This means that the apps within the pod will share the network stack and the interfaces with the host machine.
-Since the metadata service uses the pod IP for identity, rkt will _not_ register the pod with the metadata service in this mode.
-
-Note: because of the lack of the metadata service rkt does not strictly implement the app container specification in host mode.
 
 ## Private networking mode
 
-For all of the private networking options the metadata service, launched via `rkt metadata-service`, must be running.
-The service will listen on 0.0.0.0:2375 by default and provides the private networking containers the metadata services described in the App Container Spec.
```

Comment:
Should we move `The service will listen on 0.0.0.0:2375 by default and provides the metadata services described in the App Container Spec.` (with a link to the relevant section in the spec) to the getting started guide or do you think it's not important anymore?